resources/views/layouts/block.blade.php

```diff
@@ -8,7 +8,7 @@
             </p>
         </legend>
     </div>
-    <div class="col-12 col-md-7 shadow-sm">
+    <div class="col-12 {{$vertical ? 'col-md-12' : 'col-md-7'}} shadow-sm">
```

Comment:
By default, full-width markup is used. We don't need to re-install it for a specific screen.

Reply:
So this should be updated to 

`` <div class="col-12 {{!$vertical ? 'col-md-7' : ''}} shadow-sm">``

Right?